.github/workflows/oursky.yaml

```diff
@@ -35,15 +35,17 @@ jobs:
       env:
         REPO_PREFIX: ${{ secrets.AUTHGEAR_CUSTOM_BUILD_REPO_PREFIX }}
       run: |
+        MANIFEST_NAME="./$(uuidgen).json"
```

Comment:
Use 

```
MANIFEST_PATH="$(mktemp).json"
```

1. It is written to the temp directory, thus avoid the bug you encountered in the first place. (Wrote to the same location accidentally)
2. It is now named `MANIFEST_PATH`, meaning that you do not need to `custombuild/MANIFEST_PATH`